src/waitress/adjustments.py

```diff
@@ -459,9 +459,12 @@ def parse_args(cls, argv):
             else:
                 long_opts.append(opt + "=")
 
+        long_opts.append("app=")
```

Comment:
The option can’t be added to cls._params ?

Reply:
`cls._params` is for the configuration passed to `waitress.serve()` as keyword arguments. The app is an argument for `waitress.serve()`, but it's not one of the keyword arguments and requires some extra processing, and I'm trying to keep this change to a minimum. Incorporating it into `cls._params` could be worth revisiting separately though.